templates/secrets-template.yml

```diff
@@ -208,9 +196,5 @@ objects:
   metadata:
     name: kas-fleet-manager-observatorium-configuration-red-hat-sso
   stringData:
-    logs.clientId: ${OBSERVABILITY_RHSSO_LOGS_CLIENT_ID}
-    logs.clientSecret : ${OBSERVABILITY_RHSSO_LOGS_SECRET}
-    metrics.clientId: ${OBSERVABILITY_RHSSO_METRICS_CLIENT_ID}
-    metrics.clientSecret: ${OBSERVABILITY_RHSSO_METRICS_SECRET}
     grafana.clientId: ${OBSERVABILITY_RHSSO_GRAFANA_CLIENT_ID}
     grafana.clientSecret: ${OBSERVABILITY_RHSSO_GRAFANA_CLIENT_SECRET}
```

Comment:
these two are used for the control plane to read metrics using the token refresher, correct?

Reply:
Yes, we still need them.

Reply:
I am trying to determine if these can be removed (not necessarily within this PR but something to take note and create a JIRA for)

Reply:
+1 these are still needed. Since the secret no longer has any of the dataplane secrets, this can be re-purposed for only the observatorium secrets required by kfm or it can be moved to the kas-fleet-manager secret file.

Reply:
Thanks @JameelB @pb82 I am fine with the answers. Let's keep it that way for now and revisit what @JameelB said in follow up PRs